### PR TITLE
Create interactive tasks with high priority

### PR DESCRIPTION
**Pull request by ahal**

### CODE REVIEW

File: lib/ui/loaner-button.jsx [hunk 1]
```diff
@@ -112,6 +113,9 @@ let LoanerButton = React.createClass({
     // Set task,retries to 0
     task.retries = 0;
 
+    // Set high priority as a developer is waiting for it in real time
+    task.priority = 'high'
```
Comment (eliperelman) [inline lib/ui/loaner-button.jsx]: nit: missing semicolon.

**Comment by ahal** [comment]
Thanks. I'm going to close this as it doesn't actually work (see bug 1271677). I mostly just needed a diff up, and in hindsight, should have done a "compare" on my fork, rather than a pull request. Sorry.

**Comment by jonasfj** [comment]
Too late I think we should consider taking this...

Well, depending on what we do with `queue:priority:high`, if we give to everybody we should just set it here...

Otherwise, maybe we could deduce a way to determine if people have the `queue:priority:high` scope.

**Comment by jonasfj** [comment]
Discussion on [bug 1271677](https://bugzilla.mozilla.org/show_bug.cgi?id=1271677) to see if we open the scope or not...

**Comment by djmitche** [comment]
I'll close this since discussion in the bug has stalled.

**Pull request closed without merging**